playwright/_impl/_locator.py

```diff
@@ -0,0 +1,454 @@
+            strict=True,
+            **params,
+        )
+
+    async def inner_html(self, timeout: float = None) -> str:
+        params = locals_to_params(locals())
+        return await self._frame.inner_html(
+            self._selector,
+            strict=True,
+            **params,
+        )
+
+    async def inner_text(self, timeout: float = None) -> str:
+        params = locals_to_params(locals())
+        return await self._frame.inner_text(
+            self._selector,
+            strict=True,
+            **params,
+        )
+
+    async def input_value(self, timeout: float = None) -> str:
+        params = locals_to_params(locals())
+        return await self._frame.input_value(
+            self._selector,
+            strict=True,
+            **params,
+        )
+
+    async def is_checked(self, timeout: float = None) -> bool:
+        params = locals_to_params(locals())
+        return await self._frame.is_checked(
+            self._selector,
+            strict=True,
+            **params,
+        )
+
+    async def is_disabled(self, timeout: float = None) -> bool:
+        params = locals_to_params(locals())
+        return await self._frame.is_disabled(
+            self._selector,
+            strict=True,
+            **params,
+        )
+
+    async def is_editable(self, timeout: float = None) -> bool:
+        params = locals_to_params(locals())
+        return await self._frame.is_editable(
+            self._selector,
+            strict=True,
+            **params,
+        )
+
+    async def is_enabled(self, timeout: float = None) -> bool:
+        params = locals_to_params(locals())
+        return await self._frame.is_editable(
+            self._selector,
+            strict=True,
+            **params,
+        )
+
+    async def is_hidden(self, timeout: float = None) -> bool:
+        params = locals_to_params(locals())
+        return await self._frame.is_hidden(
+            self._selector,
+            strict=True,
+            **params,
+        )
+
+    async def is_visible(self, timeout: float = None) -> bool:
+        params = locals_to_params(locals())
+        return await self._frame.is_visible(
+            self._selector,
+            strict=True,
+            **params,
+        )
+
+    async def press(
+        self,
+        key: str,
+        delay: float = None,
+        timeout: float = None,
+        noWaitAfter: bool = None,
+    ) -> None:
+        params = locals_to_params(locals())
+        return await self._frame.press(self._selector, strict=True, **params)
+
+    async def screenshot(
+        self,
+        timeout: float = None,
+        type: Literal["jpeg", "png"] = None,
+        path: Union[str, pathlib.Path] = None,
+        quality: int = None,
+        omitBackground: bool = None,
+    ) -> bytes:
+        params = locals_to_params(locals())
+        return await self._with_element(
+            lambda h, o: h.screenshot(), strict=True, **params
```

Comment:
You should pass arguments to h.screenshot()

Reply:
Good catch!